libxmu will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxmu it will fail in the following way:

```
2024/11/12 23:12:52 INFO running step "test/pkgconf"
2024/11/12 23:12:52 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:52 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:52 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:52 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:52 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + basename /home/build/melange-out/libxmu-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + dev_pkg=libxmu-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + apk info -L libxmu-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + grep -q ^Name: usr/lib/pkgconfig/xmu.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + basename usr/lib/pkgconfig/xmu.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + lib_name=xmu uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + echo usr/lib/pkgconfig/xmu.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + grep -q '^usr/lib/pkgconfig/xmu.pc$\|^usr/share/pkgconfig/xmu.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + pkgconf --exists xmu uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN + pkgconf --print-errors --exists xmu uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN Package xproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN Perhaps you should add the directory containing `xproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN Package 'xproto', required by 'xmu', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN Package 'x11', required by 'xmu', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN Package 'xproto', required by 'xt', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 WARN Package 'x11', required by 'xt', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:52 INFO pod eea81c4e98c3750f9d5697078aa69c4d5678dbf5785befc8d451ec0a47360373 terminated
2024/11/12 23:12:52 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:52 INFO   workspace dir: /home/user/tmp/melange-workspace-1831763596
2024/11/12 23:12:52 INFO   guest dir: /home/user/tmp/melange-guest-1894118140
2024/11/12 23:12:52 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxmu] Error 1
```

--- a/libxmu.yaml
+++ b/libxmu.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxmu
   version: 1.2.1
-  epoch: 0
+  epoch: 1
   description: X11 miscellaneous micro-utility library
   copyright:
     - license: MIT

--- a/libxmu.yaml
+++ b/libxmu.yaml
@@ -49,10 +49,15 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - libx11-dev
         - libxmu
         - libxt-dev
         - util-linux-dev
+        - xorgproto
     description: libxmu dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxmu-doc
     pipeline:

